### PR TITLE
[New] Added test cases for pages task list

### PR DESCRIPTION
**Pull request by Srinivas9933**
Added test cases for pages task list component.

- container.spec.js

### CODE REVIEW

File: app/javascript/components/pages/task-list/container.spec.js [hunk 1]
```diff
@@ -0,0 +1,509 @@
...
+import { DashboardChip } from "../../dashboard";
+import { ListHeaderRecord } from "../../user/records";
+import { FieldRecord, FormSectionRecord } from "../../record-form/records";
+
+import TaskList from "./container";
+
+describe("<TaskList />", () => {
+  let stubI18n = null;
+  let component;
+stubI18n = stub(window.I18n, "t").withArgs("date.formats.default").returns("%d-%b-%Y");
```
Comment (jtoliver-quoin) [inline app/javascript/components/pages/task-list/container.spec.js:14]: Tab the line over

File: app/javascript/components/pages/task-list/container.spec.js [hunk 2]
```diff
@@ -0,0 +1,509 @@
...
+
+  it("should render tasks table", () => {
+    mountedComponent(
+        <TaskList />,
+        fromJS({
```
Comment (dhernandez-quoin) [inline app/javascript/components/pages/task-list/container.spec.js:19]: Create a constant at the top to reuse this data in the tests.

File: app/javascript/components/pages/task-list/container.spec.js [hunk 3]
```diff
@@ -0,0 +1,509 @@
...
+                    { id: "b", display_text: { en: "Service b" } }
+                  ]
+                }
+              ]
+            }
+          }
+        })
+      );
+    expect(screen.getAllByText(/record_id_display/i)).toHaveLength(4)
+    expect(screen.getAllByRole('row')).toHaveLength(5)
```
Comment (dhernandez-quoin) [inline app/javascript/components/pages/task-list/container.spec.js:507]: This does not seem right. Looking at the old test we should be trying to find specific task types.

**Review by dhernandez-quoin — CHANGES_REQUESTED** [review]

File: app/javascript/components/pages/task-list/container.spec.js (hunk 3 above)
Comment (Srinivas9933) [reply app/javascript/components/pages/task-list/container.spec.js]: Done

File: app/javascript/components/pages/task-list/container.spec.js (hunk 2 above)
Comment (Srinivas9933) [reply app/javascript/components/pages/task-list/container.spec.js]: Done

File: app/javascript/components/pages/task-list/container.spec.js (hunk 1 above)
Comment (Srinivas9933) [reply app/javascript/components/pages/task-list/container.spec.js]: Done

**Pull request merged**